typescript/src/entities/theme.test.ts

```diff
@@ -5,3 +5,34 @@ test('can make Theme', () => {
   const theme = new merchi.Theme();
   expect(theme).toBeTruthy();
 });
+
+test('can be activated', () => {
+  const merchi = new Merchi();
+  const theme = new merchi.Theme();
+  expect(theme.canBeActivated).toThrow();
+  theme.mainCssStatus = 3;
```

Comment:
Suggest to use constant variables instead of magic numbers

The guy I hated most in common code, recommanded me to use constant variable instead of magic numbers.

I know your point constant variable change should cause a testcase fail to indicate potentially there is a protocol change.

But I think the suggestion that guy made, is reasonable,.

Reply:
i am not sure what kind of constant you are suggesting.

the constant is already defined, it's the ones in ` src/constants/theme_status.ts`.
importing that here is a bad idea, since you are no longer testing the code, just running it (i.e. you cannot rely on the code you are testing to define your test!).

or, are you suggesting creating a variable just only within this test, as in `const testValue = 3;`. that doesn't cause any problem with making a bad tests, but, what actually does it achieve? not everything needs a name, right?